app/views/chef_profiles/show.html.erb

```diff
@@ -8,5 +8,5 @@
   <dt>Location</dt>
   <dd><%= @chef_profile.location %></dd>
 </dl>
-<%= link_to 'back to all chefs',  chef_profiles_path %>
+<%= link_to 'back to my chef profiles',  chef_profiles_path %>
```

Comment:
Doesn't this take the user back to the chef_profiles index? Not just the ones that they created?